app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
Got curious if ContainerManager::OrchestrationStack relation is also affected.
ContainerManager doesn't `include HasManyOrchestrationStackMixin`, but if it did:
```
[2] pry(main)> ems.orchestration_stacks.to_sql
=> "SELECT \"orchestration_stacks\".* FROM \"orchestration_stacks\" WHERE \"orchestration_stacks\".\"type\" IN ('ManageIQ::Providers::ContainerManager::OrchestrationStack', 'ManageIQ::Providers::Openshift::ContainerManager::OrchestrationStack') AND \"orchestration_stacks\".\"ems_id\" = 3"
```
Strange, this correctly includes the openshift descendants so seems harmless. Why doesn't the templates query?

```
[5] pry(main)> ManageIQ::Providers::ContainerManager::ContainerTemplate.descendants
=> []
```
... some false steps later ...
```
[13] pry(main)> ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate
=> ManageIQ::Providers::ContainerManager::ContainerTemplate(...
```
**OMG, the provider-specific subclasses are not loaded!  Const lookup finds this class so no autoload.**
```
[16] pry(main)> require 'manageiq/providers/openshift/container_manager/container_template.rb'
=> true
[17] pry(main)> ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate
=> ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate(...
[18] pry(main)> ManageIQ::Providers::ContainerManager::ContainerTemplate.descendants
=> [ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate(...)]
```

=> Gonna add `require_nested`.